src/modules/admin/menu/console/MenuFormSection.tsx

```diff
@@ -20,10 +31,16 @@ type FieldType = {
 const MenuFormSection: React.FC = () => {
   const { consoleSectionMode, editMenuId, changeToCategoryMode } =
     useConsoleSectionMode();
+  const { token } = theme.useToken();
   const [form] = Form.useForm();
   const [imageURL, setImageURL] = useState<string>(
     "https://source.unsplash.com/random/?food&plate&11",
   );
+  const [published, setPublished] = useState<boolean>(true);
+
+  const [openCancelModal, setOpenCancelModal] = useState<boolean>(false);
+  const [openDeleteModal, setOpenDeleteModal] = useState<boolean>(false);
+  const [loadingDelete, setLoadingDelete] = useState<boolean>(false);
```

Comment:
it fine but no need for explicit type useState can infer type from initial value